nixos/modules/services/databases/postgresql.md

```diff
@@ -207,15 +207,15 @@ For an upgrade, a script like this can be used to simplify the process:
       export NEWBIN="${newPostgres}/bin"
```

Comment:
At least for pg_anon you need the extensions here as well.
Though this was IIRC the case where you also need specify `shared_preload_libraries` yourself and this is maybe worth a mention and doesn't really need to be generalized here, does it?

Reply:
Oopsie, sorry and thanks for pointing this out! :see_no_evil: